sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/CaseDirectoryPage.java

```diff
@@ -22,6 +22,17 @@
 
 public class CaseDirectoryPage {
   public static final By NEW_CASE_BUTTON = By.cssSelector("div#caseNewCase");
+  public static final By CASE_DIRECTORY_DETAILED_RADIOBUTTON =
+      By.cssSelector("div#casesViewSwitcher span:nth-child(2) > label");
+  public static final By CASE_DIRECTORY_DETAILED_PAGE_FILTER_INPUT =
+      By.cssSelector("input#nameUuidEpidNumberLike");
+  public static final By CASE_DIRECTORY_DETAILED_PAGE_APPLY_FILTER_BUTTON =
+      By.cssSelector("div#actionApplyFilters");
+  public static final By CASE_DETAILED_TABLE_DATA = By.tagName("td");
+  public static final By CASE_DETAILED_COLUMN_HEADERS =
+      By.cssSelector("thead .v-grid-column-default-header-content");
+  public static final By CASE_DETAILED_FIRST_TABLE_ROW =
```

Comment:
should be ROWS if we're using get elements and get more than one, even if for test purpose we're taking the first one

Reply:
:) in a previous PR I had a comment to call it first row... will do it. :)